libs/ui/elements/src/lib/markdown-parser/markdown-parser.component.css

```diff
@@ -0,0 +1,271 @@
+/** Body **/
+.markdown-body {
+  -ms-text-size-adjust: 100%;
+  -webkit-text-size-adjust: 100%;
+  margin: 0px 0px 1.5rem 0px;
+  color: var(--color-main);
+  font-size: 16px;
+  line-height: 1.5;
+  word-wrap: break-word;
+}
+
+/** Emphasis **/
+
+.markdown-body strong {
+  font-weight: 600;
+  color: var(--color-secondary-darker);
+}
+
+/** Headings **/
+
+.markdown-body h1,
+.markdown-body h2,
+.markdown-body h3,
+.markdown-body h4,
+.markdown-body h5,
+.markdown-body h6 {
+  margin-top: 24px;
+  margin-bottom: 16px;
+  font-weight: 600;
+  line-height: 1.25;
+}
+
+.markdown-body h1 {
+  margin: 0.67em 0;
+  font-weight: 600;
+  padding-bottom: 0.3em;
+  font-size: 2em;
+  color: var(--color-primary);
+}
```

Comment:
The title styles currently are inconsistent with other titles, see:
![image](https://github.com/geonetwork/geonetwork-ui/assets/10629150/8196d772-dbd6-4550-8202-d22563f4773c)

I would suggest using `@apply text-title font-title` for all `hx` classes above